BankCOR/src/edu/ccsu/cs505/accounts/CreditAccount.java

```diff
@@ -0,0 +1,112 @@
+package edu.ccsu.cs505.accounts;
+import java.text.NumberFormat;
+import edu.ccsu.cs505.insufficientfunds.*;
+
+/**
+ * 
+ * This class deducts the max amount of each account in the chain until the amount is satisfied.
+ * 
+ * @author Joseph Tanasi
+ */
+
+public class CreditAccount extends Account
+{
+	private double creditLimit;
+	private static double MAX;
```

Comment:
Note by making this MAX static it would mean all credit cards for everybody in the system has the same credit limit on all credit cards (i.e. every credit card in the world has the same credit limit).  Because of this you wouldn't want to make the max a static variable.  Also an attribute should not be all caps unless the value is marked as final i.e. it is a constant.